src/blob/BlobConfiguration.ts

```diff
@@ -60,7 +61,8 @@ export default class BlobConfiguration extends ConfigurationBase {
       key,
       pwd,
       oauth,
-      disableProductStyleUrl
+      disableProductStyleUrl,
+      isBlobVersioningEnabled
```

Comment:
We might should get the "isBlobVersioningEnabled" from account model, instead of input parameter. 

We can implement how to set the value to account model later.